ci/fireci/fireciplugins/macrobenchmark.py

```diff
@@ -78,35 +92,39 @@ def _artifact_key_version(artifact):
   return f'{group_id}:{artifact_id}', version
 
 
-async def _parse_config_yaml():
-  with open('health-metrics/macrobenchmark/config.yaml') as yaml_file:
-    return yaml.safe_load(yaml_file)
+async def _process_config_yaml():
+  with open('health-metrics/benchmark/config.yaml') as yaml_file:
+    config = yaml.safe_load(yaml_file)
+    for app in config['test-apps']:
+      app['plugins'] = app.get('plugins', [])
+      app['traces'] = app.get('traces', [])
+      app['plugins'].extend(config['common-plugins'])
+      app['traces'].extend(config['common-traces'])
+
+    # Adding an empty android app for baseline comparison
+    config['test-apps'].insert(0, {'sdk': 'baseline', 'name': 'baseline'})
```

Comment:
Since we are now measuring Firebase traces only, do we still need this baseline to be present?

Reply:
Makes sense. Removed it and the reference to it in the README.